src/components/Notifications/TimeConfig.tsx

```diff
@@ -52,27 +60,25 @@ export const TimeConfigComponent: React.FunctionComponent = () => {
         setShowCustomSelect(true);
     }, []);
 
-    const handleTimePrefSelect = React.useCallback(() => {
-        setTimeSelect(timeSelect);
+    const handleTimePrefSelect = React.useCallback((time) => {
```

Comment:
Here we actually get the time as a parameter. It is [documented](https://patternfly-react.surge.sh/components/time-picker#timepicker) in the patternfly API.

![image](https://github.com/aferd/notifications-frontend/assets/3845764/79b6bb74-011f-46ff-bc77-3734b1d66e54)